crates/client/db/src/rocksdb_options.rs

```diff
@@ -0,0 +1,73 @@
+#![allow(clippy::identity_op)] // allow 1 * MiB
+#![allow(non_upper_case_globals)] // allow KiB/MiB/GiB names
+
+use crate::{contract_db, Column};
+use anyhow::{Context, Result};
+use rocksdb::{DBCompressionType, Env, Options, SliceTransform};
+
+const KiB: usize = 1024;
+const MiB: usize = 1024 * KiB;
+const GiB: usize = 1024 * MiB;
+
+pub fn rocksdb_global_options() -> Result<Options> {
+    let mut options = Options::default();
+    options.create_if_missing(true);
+    options.create_missing_column_families(true);
+    let cores = std::thread::available_parallelism().map(|e| e.get() as i32).unwrap_or(1);
+    options.increase_parallelism(cores);
+    options.set_max_background_jobs(cores);
+
+    options.set_atomic_flush(true);
+    options.set_max_subcompactions(cores as _);
+
+    options.set_max_log_file_size(10 * MiB);
+    options.set_max_open_files(2048);
+    options.set_keep_log_file_num(3);
+    options.set_log_level(rocksdb::LogLevel::Warn);
```

Comment:
Why warn and not info?

Reply:
this is not a change, it's marked as a change because git bad
 
info is the default, and it will print column stats every so often, making huuuge log files with absolutely useless information

see #345 where it was introduced